pkg/types/types.go

```diff
@@ -2,26 +2,16 @@ package types
 
 import (
 	"github.com/containernetworking/cni/pkg/types"
-	"github.com/intel/sriov-cni/pkg/dpdk"
 )
 
-// VfInformation holds VF specific informaiton
-type VfInformation struct {
-	PCIaddr string `json:"pci_addr"`
-	Pfname  string `json:"pfname"`
-	Vfid    int    `json:"vfid"`
-}
-
 // NetConf extends types.NetConf for sriov-cni
 type NetConf struct {
 	types.NetConf
-	DPDKMode   bool
-	Sharedvf   bool
-	DPDKConf   *dpdk.Conf     `json:"dpdk,omitempty"`
-	CNIDir     string         `json:"cniDir"`
-	Master     string         `json:"master"`
-	L2Mode     bool           `json:"l2enable"`
-	Vlan       int            `json:"vlan"`
-	DeviceID   string         `json:"deviceID"`
-	DeviceInfo *VfInformation `json:"deviceinfo,omitempty"`
+	DPDKMode    bool `json:"dpdkMode,omitempty"`
```

Comment:
It would be nice to not need to have differences in the pod for Mellanox and Intel NICs, couldn't it be determined purely based on if the network device is present, that way it keeps a consistent pod definition.

Reply:
+1. I thought the all idea was to deprecate the dpdk spacifc config.

Reply:
Thanks for your feedback. I think we have some other comments as well suggesting similar approach. I will work on an update to detect dpdk drivers and the `"dpdkMode"` field will be used internally by the plugin.